e2e/test/scenarios/embedding/embedding-linked-filters.cy.spec.js

```diff
@@ -73,16 +75,16 @@ describe("scenarios > embedding > dashboard > linked filters (metabase#13639, me
 
       cy.location("search").should("eq", "?state=AK&city=");
 
-      getXAxisValues()
-        .should("have.length", 1)
-        .and("contain", "AK")
+      echartsContainer()
+        .get("text")
+        .should("contain", "AK")
         .and("not.contain", "TX");
 
-      cy.get(".bar").should("have.length", 1).realHover();
+      chartPathWithColor("#509EE3").should("have.length", 1).realHover();
 
       popover().within(() => {
         testPairedTooltipValues("STATE", "AK");
-        testPairedTooltipValues("Count", "68");
+        testPairedTooltipValues("count", "68");
```

Comment:
Why should this be changed to lowercased count? If previously tooltip showed capitalized count and now it does not this is a regression we need to fix